uber/models.py

```diff
@@ -1146,6 +1149,10 @@ def badge_cost(self):
         else:
             return c.get_attendee_price(registered)
 
```

Comment:
This is the first time we've implemented a negative ``cost_property`` but it works just as you'd expect.